meerkat_frontend/templates/explore/index.html

```diff
@@ -139,22 +139,22 @@
 {% block pageJS %}
 
 <script type="text/javascript" >
-addThrobber();
+ addThrobber();
```

Comment:
Some strange indentation in this file.  Would you be able to do a find and replace on \t tab characters and then line everything up to hard 4space tabs like in python files. Then we should probably also get rid of all these single spaces that your editor has added, so that non-nested javascript lines up with column 1 of the document.  I'm sorry that I'm essentially asking you to clean up my mess - not the most exciting work - but would be good if we could keep pushing forwards with neatening the code up bit by bit.

Reply:
np, I think my editor lacked some formatting rules, this time it should be pretty!

Reply:
I removed this one-space indentation.I tried Auto indent and it almost worked, but sadly it doesn't realise that jinja needs `{{` to work properly. Have you found a way around it?